gvsbuild/projects/__init__.py

```diff
@@ -0,0 +1,85 @@
+# flake8: noqa
+
+from gvsbuild.projects.adwaita_icon_theme import Project_adwaita_icon_theme
```

Comment:
Soo... feel free to ignore me if this is too much yak shaving, but since now projects are in their own module, should we just rename them from `Project_adwaita_icon_theme` to simply `adwaita_icon_theme` or and refer to it as `projects.adwaita_icon_theme`?

Reply:
Hi @pbor, good idea. I updated the PR and used the proper PascalCase for the class names.